packages/walletconnect/__tests__/utils/common.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable @typescript-eslint/no-unsafe-return */
```

Comment:
I don't feel strongly about this so if it's too much work just ignore this, but it might be nice to fix some of these typescript errors? Again, not a big deal.